packages/react-native-win32/just-task.js

```diff
@@ -70,8 +70,8 @@ task('ts', () => {
 });
 task('clean', () => {
   return cleanTask(
-    ['dist', 'flow', 'flow-typed', 'jest', 'Libraries', 'RNTester', 'lib'].map(
```

Comment:
>flow [](start = 22, length = 4)

We haven't been doing anything with `flow-typed` or `lib` for quite a while, as neither have existed since we're deforked at the very least. Cleaned these up.